pkg/minikube/command/command_runner.go

```diff
@@ -75,6 +75,10 @@ type Runner interface {
 	// Copy is a convenience method that runs a command to copy a file
 	Copy(assets.CopyableFile) error
 
+	// CopyCheck is a convenience method that runs a command to copy a file,
+	// optionally after first checking if the file already exists.
+	CopyCheck(assets.CopyableFile, bool) error
```

Comment:
The behavior isn't obvious from the function name, and especially from the added bool. It would be more obvious if we could name this function something like `CopyIfMissing`, and have callers switch between `Copy` and `CopyIfMissing` based on their own demands.

Do you mind making this method clearer so that folks don't have to find this string to understand the behavior?

Reply:
I can revert those changes from this PR, it was just a minor optimization to avoid checking for a temporary filename